File: src/mysqlcookbook/recipes/tblmgmt/uniq_name.py
#!/usr/bin/python3
# uniq_name.py: show how to use PID to create table name

#@ _GENERATE_NAME_WITH_PID_1_
import os
#@ _GENERATE_NAME_WITH_PID_1_

pid = os.getpid()
print("PID: %s" % pid)
#@ _GENERATE_NAME_WITH_PID_2_
tbl_name = "tmp_tbl_%d" % os.getpid()
#@ _GENERATE_NAME_WITH_PID_2_
print("Table name: %s" % tbl_name)

File: src/mysqlcookbook/recipes/tblmgmt/uniq_name.rb
#!/usr/bin/ruby
# uniq_name.rb: show how to use PID to create table name

puts "PID: " + Process.pid.to_s
#@ _GENERATE_NAME_WITH_PID_
tbl_name = "tmp_tbl_" + Process.pid.to_s
#@ _GENERATE_NAME_WITH_PID_
puts "Table name: #{tbl_name}"
